### PR TITLE
BaSP-TG-126: Standardize error error messages

### PR DESCRIPTION
**Pull request by GonzaloAsencio**
- Messages for all entities must follow the same criteria.
- In the attached file are the messages agreed by the team

### CODE REVIEW

**Review by fede230298 — APPROVED** [review]

File: src/Components/Employees/validation.js [hunk 1]
```diff
@@ -3,55 +3,57 @@ import Joi from 'joi';
...
-      'string.base': `"Password" should be a type of 'text'`,
-      'string.alphanum': `"Password" should be letters and numbers`,
-      'string.empty': `"Password" cannot be an empty field`,
-      'string.pattern.name': `"Password" should be letters, numbers and minimum 8 characters`,
-      'any.required': `"Password" is a required field`
+      'string.base': 'Password must be of type text',
+      'string.alphanum': 'Password cannot contain special characters',
+      'string.min': 'Password Length must be at least 8 characters long',
+      'string.empty': 'Password cannot be an empty field',
+      'string.pattern.base': `"Password" should be letters, numbers and minimum 8 characters`,
```
Comment (ProDIGGY-vinilos) [inline src/Components/Employees/validation.js:56]: you missed to delete "" in password

Comment (GonzaloAsencio) [reply src/Components/Employees/validation.js:56]: Done in "TG-126:corrections-in-admin-employee-project"

**Review by ProDIGGY-vinilos — APPROVED** [review]
god

**Review by MaicolFreire001 — APPROVED** [review]

File: src/Components/Admins/validations.jsx [hunk 2]
```diff
@@ -7,36 +7,36 @@ export const schema = Joi.object({
     .pattern(/^([^0-9]*)$/i)
     .required()
     .messages({
-      'string.alphanum': 'Name cannot contain special characters.',
-      'string.min': 'Name Length must be at least 3 characters long.',
-      'string.pattern.base': 'Name must have only letters.',
-      'string.empty': 'Name cannot be an empty field.',
-      'any.required': 'Name is requiered.'
+      'string.alphanum': 'Name cannot contain special characters',
+      'string.min': 'Name Length must be at least 3 characters long',
```
Comment (JuliMonte5) [inline src/Components/Admins/validations.jsx:11]: "Length" is not part of the standar message, delete it

File: src/Components/Admins/validations.jsx [hunk 3]
```diff
@@ -7,36 +7,36 @@ export const schema = Joi.object({
...
+      'string.empty': 'Email cannot be an empty field',
+      'any.required': 'Email is requiered'
     }),
   password: Joi.string().alphanum().min(8).required().messages({
-    'string.alphanum': 'Password cannot contain special characters.',
-    'string.min': 'Password Length must be at least 8 characters long.',
-    'string.empty': 'Password cannot be an empty field.',
-    'any.required': 'Password is requiered.'
+    'string.alphanum': 'Password cannot contain special characters',
+    'string.min': 'Password Length must be at least 8 characters long',
```
Comment (JuliMonte5) [inline src/Components/Admins/validations.jsx:38]: same as 11

File: src/Components/Employees/validation.js [hunk 4]
```diff
@@ -3,55 +3,57 @@ import Joi from 'joi';
...
     .required()
     .messages({
-      'string.base': `"Password" should be a type of 'text'`,
-      'string.alphanum': `"Password" should be letters and numbers`,
-      'string.empty': `"Password" cannot be an empty field`,
-      'string.pattern.name': `"Password" should be letters, numbers and minimum 8 characters`,
-      'any.required': `"Password" is a required field`
+      'string.base': 'Password must be of type text',
+      'string.alphanum': 'Password cannot contain special characters',
+      'string.min': 'Password Length must be at least 8 characters long',
```
Comment (JuliMonte5) [inline src/Components/Employees/validation.js:54]: same as 11

File: src/Components/Employees/validation.js (hunk 1 above)
Comment (JuliMonte5) [reply src/Components/Employees/validation.js:56]: I'll mark it as unresolved for now

File: src/Components/TimeSheets/Form/validations.js [hunk 5]
```diff
@@ -7,13 +7,13 @@ export const timeSheetValidation = Joi.object({
     .regex(/^(?=.*[a-zA-Z].*)([\w\s\W]+)$/)
     .min(3)
     .messages({
-      'string.empty': 'Description is required',
-      'string.pattern.base': 'Description should contain at least one letter',
-      'string.min': 'Description should have at least 3 characters',
+      'string.empty': 'Description cannot be an empty field',
+      'string.pattern.base': 'Description must be at least 1 letter',
+      'string.min': 'Description must be at least 3 characters long	',
```
Comment (JuliMonte5) [inline src/Components/TimeSheets/Form/validations.js:12]: delete extra spaces at the end of the message

File: src/Components/Employees/validation.js [hunk 6]
```diff
@@ -3,55 +3,57 @@ import Joi from 'joi';
...
-      'string.base': `"Name" should be a type of 'text'`,
-      'string.alphanum': `"Name" should be only characters`,
-      'string.empty': `"Name" cannot be an empty field`,
-      'string.pattern.name': `"Name" should be only letters`,
-      'any.required': `"Name" is a required field`
+      'string.base': 'Name must be of type text',
+      'string.alphanum': 'Name cannot contain special characters',
+      'string.empty': 'Name cannot be an empty field',
+      'string.pattern.base': 'Name must be of type text',
+      'any.required': 'Name is required'
```
Comment (JuliMonte5) [inline src/Components/Employees/validation.js:13]: Shouldn't Name have the same .min() validation as Admin?

File: src/Components/SuperAdmins/Form/validations.js [hunk 7]
```diff
@@ -7,40 +7,40 @@ const schema = Joi.object({
...
     .empty()
     .messages({
-      'string.alphanum': 'Name cannot contain special characters.',
-      'string.pattern.base': 'Name can only contain letters.',
-      'string.empty': 'Name cannot be an empty field.',
-      'any.required': 'Name is requiered.'
+      'string.alphanum': 'Name cannot contain special characters',
+      'string.pattern.base': 'Name must be of type text',
+      'string.empty': 'Name cannot be an empty field',
+      'any.required': 'Name is requiered'
```
Comment (JuliMonte5) [inline src/Components/SuperAdmins/Form/validations.js:13]: Same as 13

**Review by sofihoj — APPROVED** [review]

File: src/Components/Admins/validations.jsx (hunk 2 above)
Comment (GonzaloAsencio) [reply src/Components/Admins/validations.jsx:11]: done

File: src/Components/Admins/validations.jsx (hunk 3 above)
Comment (GonzaloAsencio) [reply src/Components/Admins/validations.jsx:38]: Done

File: src/Components/Employees/validation.js (hunk 4 above)
Comment (GonzaloAsencio) [reply src/Components/Employees/validation.js:54]: Done

File: src/Components/TimeSheets/Form/validations.js (hunk 5 above)
Comment (GonzaloAsencio) [reply src/Components/TimeSheets/Form/validations.js:12]: Done

File: src/Components/Employees/validation.js (hunk 6 above)
Comment (GonzaloAsencio) [reply src/Components/Employees/validation.js:13]: I agree, just add the corresponding text to each validation

Comment (GonzaloAsencio) [reply src/Components/Employees/validation.js:13]: done

File: src/Components/SuperAdmins/Form/validations.js (hunk 7 above)
Comment (GonzaloAsencio) [reply src/Components/SuperAdmins/Form/validations.js:15]: Done

**Pull request merged**